src/app/models/task.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface ITask {
```

Comment:
Why ITask? I couldn't get the reference for the name 'ITask'.

Reply:
Interfarce naming convention Apu ( [resource](https://dev.to/adammc331/interface-naming-conventions-3m0o) )

Reply:
Changed ITask to Task.